rotkehlchen/accounting/structures/processed_event.py

```diff
@@ -117,6 +123,21 @@ def to_exported_dict(
                 exported_dict['asset'] = ''
             if tx_hash is not None:
                 exported_dict['notes'] = f'{evm_explorer}{tx_hash}  ->  {self.notes}'
+            if self.location in EVM_LOCATIONS and database is not None:
+                def _maybe_add_label_with_address(matched_address: re.Match[str]) -> str:
+                    """Adds a label of the address if it exists in a given string"""
+                    chain_address = ChainAddress(
+                        address=string_to_evm_address(matched_address.group(0)),
```

Comment:
group(0) can raise an error if for some reason the address doesn't appear right? Can you check that? also I believe you need to check if it is none or not before creating the chainAddress?

Reply:
I tested with the cases where nothing matches with the regex or multiple matches are found. It works in both cases because the function `_maybe_add_label_with_address` is called only when something is matched, so inside the function, it always returns the matched string. Also, the default argument of `.group()` is 0, so removing that.